dbt/adapters/athena/utils.py

```diff
@@ -9,6 +11,19 @@ def clean_sql_comment(comment: str) -> str:
     return " ".join(line for line in split_and_strip if line)
 
 
+def stringify_table_parameter_value(value: Any) -> str:
+    """Convert any variable to string for Glue Table property."""
+    if isinstance(value, (dict, list)):
+        value_str: str = json.dumps(value)
+    else:
+        value_str = str(value)
+    return value_str[:512000]
+
+
+def is_valid_table_parameter_key(key: str) -> bool:
+    return len(key) <= 255 and bool(re.match(r"^[\u0020-\uD7FF\uE000-\uFFFD\t]*$", key))
```

Comment:
Could you add a little comment on what the regex does? I believe that can be helpful in the future.